src/components/cart/cartNav.js

```diff
@@ -0,0 +1,86 @@
+import { Link } from "react-router-dom";
```

Comment:
One suggestion, you can have one Navigation header for all pages. Since there is not much change in the navigation component across different pages.